testing/test_pluginmanager.py

```diff
@@ -20,8 +19,19 @@ def pytestpm():
     return PytestPluginManager()
 
 
+@pytest.fixture
+def config_for_test():
+    from _pytest.config import get_config
+
+    config = get_config()
+    try:
+        yield config
+    finally:
+        config._ensure_unconfigure()  # cleanup, e.g. capman closing tmpfiles.
```

Comment:
Would be good to share the fixture, but there is no conftest.. :D - any advice?

Reply:
I think we can move that to `pytester` perhaps?

Reply:
Ok. Moved to `_config_for_test` therein.

Reply:
Good idea to use `_`. 👍